src/spootnik/reporter.clj

```diff
@@ -246,19 +246,14 @@
       (tmr/stop (tmr/timer registry (->alias alias)))))
   SentrySink
   (capture! [this e]
-    (error e "captured exception")
-    (when raven
-      (try
-        (raven/capture! {:http raven} (:dsn sentry) e {})
-        (catch Exception e
-          (error e "could not send capture")))))
+    (capture! this e {}))
   (capture! [this e tags]
-    (error e "captured exception")
-    (when raven
-      (try
-        (raven/capture! {:http raven} (:dsn sentry) e tags)
-        (catch Exception e
-          (error e "could not send capture")))))
+    (when-let [event-id (raven/sentry-uuid!)] raven
+              (error e (str "captured exception as sentry event: " event-id))
+              (try
+                (raven/capture! {:http_client raven :event_id event-id} (:dsn sentry) e tags)
+                (catch Exception e
+                  (error e "could not send capture for event: " event-id)))))
```

Comment:
I'd prefer something along those lines

```clj
(try
  (when-let [event-id (raven/capture! ... e tags)]
    (error e (str "captured exception" event-id))
  (catch ...))
```

Reply:
even if, I don't see the reason for the `(try (catch ...))` in the first place

Reply:
Agreed.

Done :+1:

Reply:
(I removed the try/catch as well)